nixos/modules/security/acme.nix

```diff
@@ -221,82 +199,32 @@ in
                   description = "Renew ACME Certificate for ${cert}";
                   after = [ "network.target" "network-online.target" ];
                   wants = [ "network-online.target" ];
+                  wantedBy =
+                    lib.optionals config.services.nginx.enable  [ "nginx.service" ] ++ 
```

Comment:
Does it make sense that his is here?

Given that the `enableACME` option is in the nginx module, shouldn't that one use `wants` on the cert services instead?

It looks like this way, any webserver present in nixpkgs that might use these certs would have to add itself here.

If doing it the other way around is currently not feasible (e.g. because it would be somehow more difficult or inconvenient), can you add a comment stating so?

Reply:
Also see discussion above. I was planning to do this but forgot. Thanks for reminding me

Reply:
I removed these now!